.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
```

Comment:
An enhancement, as a bug report, should include some code and rationale.
Good feature requests: #120 #7895 #8045 #7992 #7677
Please add an example of feature request to the xdocs.

Reply:
You want additions to this page https://checkstyle.org/report_issue.html ?

Reply:
> You want additions to this page https://checkstyle.org/report_issue.html ?

Yes. Please provide examples for https://checkstyle.org/report_issue.html#How_to_request_new_feature_for_existing_functionality.3F
and https://checkstyle.org/report_issue.html#How_to_request_new_Check.2FModule.3F

Reply:
Done. Please provide feedback. @pbludov

Reply:
Site link is in PR description.